app/src/common/urls.js

```diff
@@ -110,6 +110,7 @@ export const URLS = {
   choiceSuggestedItems: (activeProject) => `${urlBase}${activeProject}/item/suggest/choice`,
   launchNameSearch: (activeProject) => (searchTerm = '') =>
     `${urlBase}${activeProject}/launch/names?filter.cnt.name=${searchTerm}`,
+  launchesAll: (activeProject) => `${urlBase}${activeProject}/launch/names`,
```

Comment:
Endpoint return array of launches names please rename property to something more connected to it